### PR TITLE
Normalize modals, add series edit modal

### PR DESCRIPTION
**Pull request by harshithmohan**

### CODE REVIEW

File: src/core/rtkQuery/splitV3Api/seriesApi.ts [hunk 1]
```diff
@@ -189,6 +199,33 @@ const seriesApi = splitV3Api.injectEndpoints({
         url: `Series/${seriesId}/Images`,
       }),
     }),
+
+    // Queue a refresh of the AniDB Info for series with ID
```
Comment (revam) [inline src/core/rtkQuery/splitV3Api/seriesApi.ts:203]: Copy pasta 🍝

File: src/components/Collection/Series/EditSeriesTabs/NameTab.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+import Input from '@/components/Input/Input';
+import { useGetSeriesQuery } from '@/core/rtkQuery/splitV3Api/seriesApi';
+
+type Props = {
+  seriesId: number;
+};
+
+const NameTab = ({ seriesId }: Props) => {
+  const [name, setName] = useState('');
```
Comment (revam) [inline src/components/Collection/Series/EditSeriesTabs/NameTab.tsx:12]: This doesn't look like it's hooked up yet. Are we missing an endpoint?

File: src/components/Collection/Series/EditSeriesModal.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+import ModalPanel from '@/components/Panels/ModalPanel';
+
+type Props = {
+  show: boolean;
+  onClose: () => void;
+  seriesId: number;
+};
+
+const tabs = {
+  // name: 'Name',
```
Comment (revam) [inline src/components/Collection/Series/EditSeriesModal.tsx:16]: It's also commented out.

Comment (harshithmohan) [reply src/components/Collection/Series/EditSeriesModal.tsx:16]: It's not done, that's why 🙃

File: src/components/Collection/Series/EditSeriesTabs/NameTab.tsx (hunk 2 above)
Comment (harshithmohan) [reply src/components/Collection/Series/EditSeriesTabs/NameTab.tsx:12]: yes

**Review by hidden4003 — APPROVED** [review]

File: src/components/Collection/Series/EditSeriesTabs/SeriesActionsTab.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  });
+
+  return (
+    <div className="flex grow flex-col gap-y-2">
+      <Action
+        name="Rescan Files"
+        onClick={() => {
+          rescanSeriesFiles({ seriesId })
+            .then(() => toast.success('Series files rescan queued!'))
+            .catch(() => toast.error('Series files rescan failed!'));
```
[the remark this answers is not in the document]
Comment (harshithmohan) [reply src/components/Collection/Series/EditSeriesTabs/SeriesActionsTab.tsx:47]: I just realized... let me remove that

**Pull request merged**